src/power_grid_model/manual_testing.py

```diff
@@ -8,11 +8,21 @@
 
 import json
 from pathlib import Path
-from typing import IO, Any, Dict, List, Optional, Union
+from typing import IO, Any, List, Optional, Union, cast
 
 import numpy as np
 
 from . import initialize_array
+from .data_types import (
```

Comment:
Please use absolute (explicit) imports.  I see we don't do it for other imports as well; there are a lot of relative inputs in the code. Maybe good to update all from implicit to explicit imports

Reply:
It was added in commit [336809f](https://github.com/alliander-opensource/power-grid-model/pull/106/commits/336809facc421c13150413c2a914b5c51ec35f5e)

Reply:
Why are absolute imports better?

Reply:
Relative imports are allowed, but PEP 8 recommends absolute imports: https://peps.python.org/pep-0008/